mlir/include/mlir/Dialect/MIOpen/LowerMIOpenOps.h

```diff
@@ -6650,10 +6653,105 @@ struct ThreadwiseCopyRewritePattern
         SmallVector<Value, 8> destLowerIndices =
             layeredDestIndices[layeredDestIndices.size() - 1];
 
+        // XXX: we do not  implement the logic of toEmitOOBStoreCheck yet
```

Comment:
should we use `boundCheckDestAttr` , just like how `toEmitOOBLoadCheckLogic` is done with `boundCheckSourceAttr`?